tests/test_mdlp.py

```diff
@@ -32,165 +34,31 @@ def test_find_cut_no_cut():
     k = find_cut(y, 0, len(y))
     assert_equal(-1, k)
 
-def test_mdlp_iris():
-    from sklearn.datasets import load_iris
+def test_fit_transform():
+  X = np.array([
+    [0.1, 0.1],
+    [0.2, 0.4],
+    [0.3, 0.2],
+    [0.4, 0.3]
+  ])
+  y = np.array([0, 0, 1, 2])
+  transformed = MDLP(shuffle=False).fit_transform(X, y)
+  expected = [
+    [0, 0],
+    [0, 0],
+    [1, 0],
+    [2, 0],
+  ]
+  assert_array_equal(transformed, expected)
 
-    iris = load_iris()
-    X = iris.data
-    y = iris.target
-    mdlp = MDLP(shuffle=False)
-    transformed = mdlp.fit_transform(X, y)
+  # discretization is invariant under rescaling of the data
```

Comment:
I commend you to test things like this. But 

1. I would encourage you to put it in another test file
2. I would try out different factors 

See https://github.com/scikit-learn/scikit-learn/blob/discrete/sklearn/preprocessing/tests/test_discretization.py#L173.